docs/configuration.md

```diff
@@ -77,6 +77,8 @@ Don't forget to reload `systemd` conf and restart `xo-server`:
 # systemctl restart xo-server.service
 ```
 
+**Note:** The `--use-openssl-ca` option is ignored by Node if Xen-Orchestra is run with Linux capabilities. Capabilities are commonly used to bind applications to privileged ports (<1024) (i.e. `CAP_NET_BIND_SERVICE`). Local NAT rules (`iptables`) or a reverse proxy would be required to use privileged ports and a custom certficate authority.
+
```

Comment:
Note that you can run `xo-server` as `root` and ask it to drop privileges: https://github.com/vatesfr/xen-orchestra/blob/3befaac1147ddfcea7f3ee44a15d7a6bf07844fe/packages/xo-server/sample.config.toml#L29-L43

It's not perfect but it can be used as a work-around.